tor_ocr/core/blossom.py

```diff
@@ -0,0 +1,71 @@
+import requests
+
+class ConfigurationError(Exception):
+    pass
+
+class BlossomAPI(object):
+    def __init__(
+            self,
+            email: str=None,
+            password: str=None,
+            api_key: str=None,
+            api_base_url: str="http://api.grafeas.localhost:8000",
+            login_url: str="http://grafeas.localhost:8000/login/"
+    ):
+        if not email:
+            raise ConfigurationError("Need an email address!")
+        if not password:
+            raise ConfigurationError("Need a password!")
+        if not api_key:
+            raise ConfigurationError("Need a password!")
```

Comment:
This is definitely my fault, but this error message should read "Need an API key!"

Reply:
no problem, should be fixed now